### PR TITLE
fix bug in VRT naming

### PR DESCRIPTION
**Pull request by bbuzzanga**

### CODE REVIEW

**Comment by sssangha** [comment]
@bbuzz31 @dbekaert looks good to me!

**Comment by sssangha** [comment]
@bbuzz31 @dbekaert on second though, give me a few minutes! Tracked down another minor bug. Testing now, will update accordingly

**Comment by bbuzzanga** [comment]
Would also be a good idea to by default add all products to the stack, regardless if they have tropo corrections
Currently, it will only add tropo if they are requested

**Comment by sssangha** [comment]
@bbuzz31 I'm not sure about mixing corrected and uncorrected products. If the user explicitly wishes to perform atmospheric corrections, those layers should remain segregated from the uncorrected ones.

**Comment by sssangha** [comment]
@dbekaert @bbuzz31 ok, fixed the other bug I found!

**Comment by dbekaert** [comment]
> @bbuzz31 I'm not sure about mixing corrected and uncorrected products. If the user explicitly wishes to perform atmospheric corrections, those layers should remain segregated from the uncorrected ones.

We should not really be mixing products.
Either they are tropo corrected or they are not.
Should be explicit request from user.

File: tools/ARIAtools/tsSetup.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def generateStack(aria_prod,inputFiles,outputFileName,workdir='./'):
         print('Number of unwrapped interferograms discovered: ', len(intList))
         Dlist = intList
     elif inputFiles in ['tropocorrected_products', 'tropocorrected', 'tropo']:
-        domainName = 'tropocorrected_products'
+        domainName = 'unwrappedPhase'
```
Comment (dbekaert) [inline tools/ARIAtools/tsSetup.py:117]: Ok, i understand this is needed given mintpy is not aware of us making any corrections.
Looks good.

**Review by dbekaert — APPROVED** [review]

**Pull request merged**